src/doc/trpl/compound-data-types.md

```diff
@@ -47,8 +47,8 @@ This pattern is very powerful, and we'll see it repeated more later.
 
 There are also a few things you can do with a tuple as a whole, without
 destructuring. You can assign one tuple into another, if they have the same
-contained types and arity. Tuples have the same arity when they have the same
-length.
+contained types and [arity](glossary.html#arity). Tuples have the same arity
+when they have the same length.
```

Comment:
I've been surprised when I've seen it, but maybe because that's just not seen as often. Should be fine. :)